gulpfile.js

```diff
@@ -115,7 +115,7 @@ gulp.task('scripts', ['lint'], function scriptsTask() {
   return bundledStream;
 });
 
-gulp.task('styles', function stylesTask() {
+gulp.task('styles', ['sass-lint'], function stylesTask() {
```

Comment:
@lmorchard The **gulp-sass-lint** module actually uses [**sass-lint**](https://github.com/sasstools/sass-lint) which is a "Pure Node.js Sass linting" module. So thankfully no Ruby needed.